### ISSUE TITLE
understanding about test

### ISSUE THREAD

**Issue by Feobi1999** [fanq15/FSOD-code#38]
During the test, we first get the similarity between one image and a certain (or 5) object categories every episode, and finally put together the results of multiple episodes of testing. When testing, is the selection of support images completely random or needs to be consistent with the query image? Is the support image selected from the training set or the test set?

**Comment by Feobi1999** [comment]
I don’t know if my understanding is correct, looking forward to your reply
